### PR TITLE
[idea] #174 taking no.of tiny editors in featured pane through settings panel

### LINKED ISSUE #174: Take no.of tiny editors in featured pane through settings panel.

### PR DESCRIPTION
**Pull request by prudhvije**
fixes #174

### CODE REVIEW

**Comment by prudhvije** [comment]
image

File: plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/ui/SettingsPanel.java [hunk 1]
```diff
@@ -40,6 +40,7 @@
     private static final String HELP_TEXT =
         "Please enter comma separated regex"
             + "(e.g. java.util.[A-Z][a-z0-9]*, org.slf4j.Logger)";
+    private static final String MAX_TINY_EDITORS = "Max. Tiny Editors in Featured Pane";
```
Comment (ScrapCodes) [inline plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/ui/SettingsPanel.java]: Change this string to "Featured count:"

**Comment by prudhvije** [comment]
Updated:
image

**Comment by ScrapCodes** [comment]
Why is it labeled help wanted ?

**Comment by ScrapCodes** [comment]
merged

**Pull request closed without merging**